src/en/temp-release-notes.md

```diff
@@ -585,15 +585,14 @@ when deploying individual charms. The bundle YAML can include a section
 called "bindings", defining the map of endpoint names to space names.
 
 Example bundle.yaml excerpt:
-    ...
+
```

Comment:
for the purposes of docs, it is best to code fence these so the highlighting (may) work:

```yaml
   
```